### ISSUE TITLE
Example may not work when using SDk-style project

### ISSUE THREAD

**Issue by ManfredLange** [dotnet/EntityFramework.Docs#2094]
When following this example one might decide to put the models, migrations and context in a separate assembly (class library). This separate assembly is created using the SDK-style format.

When adding migrations then using `Update-Database` may fail with `MissingManifestResourceException`. This can be resolved by adding the following to the project file of the assembly containing the migrations:

```xml
<PropertyGroup>
  <EmbeddedResourceUseDependentUponConvention>true</EmbeddedResourceUseDependentUponConvention>
</PropertyGroup>
```

For more details see this comment on github: https://github.com/dotnet/ef6/issues/1258#issuecomment-531355034

I'd like to suggest adding a note about the possible exception along with a link to the workaround. It is likely to save others several hours of frustrated search.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9c8be815-df51-dc77-fb5b-ccfa7f742e10
* Version Independent ID: 353c54e3-b7c9-fcdd-ca71-2a11beab19fd
* Content: [Code First to a New Database - EF6](https://docs.microsoft.com/en-us/ef/ef6/modeling/code-first/workflows/new-database)
* Content Source: [entity-framework/ef6/modeling/code-first/workflows/new-database.md](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/ef6/modeling/code-first/workflows/new-database.md)
* Product: **entity-framework**
* Technology: **entity-framework-6**
* GitHub Login: @divega
* Microsoft Alias: **avickers**

**Comment by bricelam** [comment]
It's a bummer they didn't fix that in 3.1

**Comment by ajcvickers** [comment]
@ManfredLange Please follow up on https://github.com/dotnet/sdk/issues/3639 if you think this is something that should be changed.
